src/redux/actions/pipeline/runGem2s.js

```diff
@@ -31,33 +31,29 @@ const runGem2s = (experimentId, paramsHash) => async (dispatch) => {
         },
         body: JSON.stringify({ paramsHash }),
       },
+      false,
     );
 
-    const json = await response.json();
-    throwIfRequestFailed(response, json, endUserMessages.ERROR_STARTING_PIPLELINE);
-
     dispatch({
       type: EXPERIMENT_SETTINGS_PIPELINE_START,
       payload: {},
     });
 
     dispatch(loadBackendStatus(experimentId));
   } catch (e) {
-    let { message } = e;
-    if (!isServerError(e)) {
-      console.error(`fetch ${url} error ${message}`);
-      message = endUserMessages.CONNECTION_ERROR;
-    }
+    let errorMessage = handleError(e, endUserMessages.ERROR_STARTING_PIPLELINE);
+
     console.log(`error run gem2s ${e}`);
     // temporarily give the user more info if the error is permission denied
-    if (message.includes('does not have access to experiment')) {
-      message += ' Refresh the page to continue with your analysis.';
+    if (errorMessage.includes('does not have access to experiment')) {
+      errorMessage += ' Refresh the page to continue with your analysis.';
```

Comment:
if the user tries to run the pipeline without permissions it will show the "It's not you it's us" page. In that page there's no way to go back to data processing (which they can indeed explore but not run). It's bad UX as it might not be immediately clear what to do. Ideally, we would prevent the user from trying to run it, but for the time being Vicky accepted telling the user to refresh the page (so they can once again explore data processing)